base_layer/mmr/src/mutable_mmr.rs

```diff
@@ -97,18 +98,25 @@ where
         (hash, deleted)
     }
 
-    /// Returns a merkle(ish) root for this merkle set.
+    /// Returns the full MMR merkle root and the pruned MMR merkle(ish) root for this merkle set.
     ///
-    /// The root is calculated by concatenating the MMR merkle root with the compressed serialisation of the bitmap
-    /// and then hashing the result.
-    pub fn get_merkle_root(&self) -> Hash {
+    /// The full MMR merkle root is the MMR root and excludes the state of the roaring bitmap.
+    /// The pruned MMR merkle(ish) root is calculated by concatenating the MMR merkle root with the compressed
+    /// serialisation of the bitmap and then hashing the result.
+    pub fn get_merkle_root(&self) -> (Hash, Hash) {
```

Comment:
I think we should leave the method as it originally was. 

As it stands the semantics of `get_mmr_root` is defined be "a full merklish commitment to the entire datastructure state", so  it should include the hash of the roaring bitmap.

If you want to obtain the hash contributors, then add methods called `get_mmr_only_root` and `get_deleted_nodes_hash` (possibly there's a better name).

Reply:
The function was changed back to the original version and a second function `get_mmr_only_root` was added.